metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
I haven't run/validated that command, but ran through the main ASF page on it:

ASF page on compatible/incompatible licenses: https://www.apache.org/legal/resolved.html

**The most obvious thing I saw was that BSD-4-Clause is explicitly not allowed.**

* MIT, Apache 2.0 - Allowed
* bsd-3-clause - Allowed
* ( bsd-3-clause OR MIT) - Both Allowed
* (BSD-3-Clause AND MIT) - Both Allowed
* ISC, BSD - Both Allowed
* **bsd-4-clause - NOT ALLOWED**
* MIT/X11 - Allowed
* bsd-2-clause - Allowed
* AFLv2.1 - Maybe allowed? 3.0 is
* (BSD-2-Clause OR MIT OR Apache-2.0) - All Allowed
* (MIT AND CC-BY-3.0) - Both Allowed
* Unlicense - ? Need info
* "SEE LICENSE IN COPYING.txt", - ? Need fino
* CC-BY-3.0 - Allowed
* CC-BY-4.0 - Allowed
* BSD-like - Maybe allowed? Need more info
* (OFL-1.1 AND MIT) - Unsure about OFL.  MIT allowed
* (BSD-2-Clause OR MIT OR Apache-2.0) - All allowed
* (WTFPL OR ISC) - Both Allowed
*(WTFPL OR MIT) - Both Allowed
* WTFPL - Allowed
* Public Domain - Possibly?  There's some discussion on the page I'd need to read further